ee/modules/110-istio/openapi/config-values.yaml

```diff
@@ -271,6 +271,122 @@ properties:
               format: int64
             value:
               type: string
+      resourcesManagement:
+        description: |
+          istiod resources management options.
+        default: {}
+        x-examples:
+        - mode: VPA
+          vpa:
+            mode: Auto
+            cpu:
+              min: "50m"
+              max: "2"
+              limitRatio: 1.5
+            memory:
+              min: "256Mi"
+              max: "2Gi"
+              limitRatio: 1.5
+        - mode: Static
+          static:
+            requests:
+              cpu: "55m"
+              memory: "256Mi"
+            limits:
+              cpu: "2"
+              memory: "2Gi"
+        properties:
+          mode:
+            type: string
+            description: |
+              The mode for managing resource requests. Classical `Static` requests/limit or `VPA`.
+            enum: ['VPA', 'Static']
+            default: 'VPA'
+          vpa:
+            type: object
+            default: {}
+            description: |
+              Parameters of the VPA mode.
+            properties:
+              mode:
+                type: string
+                description: |
+                  The VPA usage mode.
+                enum: ['Initial', 'Auto']
+                default: 'Auto'
+              cpu:
+                type: object
+                default: {}
+                description: |
+                  CPU-related parameters.
+                properties:
+                  max:
```

Comment:
As I remember we use the next line to describe CPU:
```
  max:
    oneOf:
      - type: string
        pattern: "^[0-9]+m?$"
      - type: number
 ```

because it could be set in 2 ways: `1` or `1000m`